class MyException(Exception): pass
def MyFunction(a):pass

